vnext/ReactWindowsCore/WebSocketModule.h

```diff
@@ -6,9 +6,12 @@
 #include <cxxreact/CxxModule.h>
 #include "IWebSocket.h"
 
-namespace Microsoft {
-namespace React {
+namespace Microsoft::React {
 
+///
+/// Realizes <c>NativeModules</c> projection.
```

Comment:
>Realizes <c>NativeModules</c> projection. [](start = 4, length = 41)

Enclose in <summary/> tags?

Reply:
Missed it!
Will fix in subsequent change.